scripts/ci/gitlab/pipeline/publish.yml

```diff
@@ -2,32 +2,33 @@
 # This file is part of .gitlab-ci.yml
 # Here are all jobs that are executed during "publish" stage
 
-.build-push-docker-image:
+.build-push-docker-image-common:
   extends:
-    - .build-refs
     - .kubernetes-env
+  stage:                           publish
   variables:
     CI_IMAGE:                      $BUILDAH_IMAGE
     GIT_STRATEGY:                  none
     DOCKERFILE:                    $PRODUCT.Dockerfile
-    IMAGE_NAME:                    docker.io/parity/$PRODUCT
+    IMAGE_NAME:                    docker.io/$IMAGE_PATH
   before_script:
     - cd ./artifacts/$PRODUCT/
     - VERSION="$(cat ./VERSION)"
     - echo "${PRODUCT} version = ${VERSION}"
     - test -z "${VERSION}" && exit 1
   script:
-    - test "$Docker_Hub_User_Parity" -a "$Docker_Hub_Pass_Parity" ||
+    - test "$DOCKERIO_USER" -a "$DOCKERIO_PASS" ||
```

Comment:
Please, can you use "DOCKER" so it's consistent to other pipeline configs (for example: https://github.com/paritytech/polkadot/blob/master/scripts/ci/gitlab/pipeline/publish.yml#L22)

Reply:
ok, done: [80de118](https://github.com/paritytech/substrate/pull/12675/commits/80de118a0d16328725a5e17788097e025aa41982)

Reply:
thanks!